pkgs/top-level/all-packages.nix

```
@@ -32890,7 +32890,7 @@ with pkgs;
 
   nix-eval-jobs = callPackage ../tools/package-management/nix-eval-jobs { };
 
-  nix-doc = callPackage ../tools/package-management/nix-doc { nix = nix_2_3; };
+  nix-doc = callPackage ../tools/package-management/nix-doc { inherit nix; };
```

Maybe we can drop this?

```suggestion
  nix-doc = callPackage ../tools/package-management/nix-doc { };
```

---

Yup. Thanks! Committed.